DER/ExchangeTradedDerivatives/Futures.rdf

```diff
@@ -592,8 +424,28 @@
 		<skos:definition xml:lang="en">How many of the underlying securities can be sold.</skos:definition>
 	</owl:DatatypeProperty>
 	
-	<owl:Class rdf:about="https://spec.edmcouncil.org/fibo/ontology/MD/DerivativesTemporal/FuturesTemporal/FuturesTradingAccountHolder">
-		<rdfs:subClassOf rdf:resource="&fibo-fnd-agr-ctr;Counterparty"/>
+	<owl:Class rdf:about="&fibo-fbc-fi-fi;Future">
+		<rdfs:subClassOf rdf:resource="&fibo-der-der-fwd;ForwardContract"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-etd-fut;pricingDeterminedBy"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-etd-std;DerivativesPriceDeterminationMethod"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
```

Comment:
No this does not seem right at all for a listed future instrument, which at the time it is listed (or ever) may have no holder/buyer.

Reply:
Eliminated the restriction as mentioned above, and made additional changes to eliminate further redundancy.